nodes/Assert.py

```diff
@@ -1 +1,13 @@
+"""
+Assert astroid node
+
+Attributes:
+    - test  (compare)  the condition to be evaluated True or False
```

Comment:
I'm guessing the `test` can really be an arbitrary expression, not necessarily a comparison node. For example, if you had a helper function `def f(n): return n > 5` and then did `assert f(10)`, the `test` would be a function call expression, right?